misc/dist-test-cases

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "EXTRA_DIST = misc/units \\" > "$1"
+find Tmain Units -type f | sed -e 's/$/\\/' >> "$1"
+echo '$(NULL)' >> "$1"
```

Comment:
Hum, that might show another issue: how to add a test case?  and even more so, how to test it before it gets committed?

Though, the question "how to add a test case" is relevant in all cases if we need to list is explicitly somewhere.  It can be as simple as running a script to update something, but it would have to be documented.